llvm/cmake/config-ix.cmake

```diff
@@ -258,6 +258,37 @@ if(LLVM_HAS_LOGF128)
   set(LLVM_HAS_LOGF128 "${HAS_LOGF128}")
 endif()
 
+# Check for ICU.
+if(LLVM_ENABLE_ICU)
+  set(LIBRARY_SUFFIXES ${CMAKE_FIND_LIBRARY_SUFFIXES})
+  set(CMAKE_FIND_LIBRARY_SUFFIXES ".so")
```

Comment:
No plans to support Windows?

Reply:
I hope to support as many platforms as possible. Would adding the `.dll` suffix to this list allow us to take advantage of the Windows SDK?

Reply:
Windows provides import library icu.lib (the dll is in Windows/system32) which is different with standard icu distribution and FindICU.cmake doesn't support Windows bundled ICU, so we may need some extra works here.

Reply:
It sounds like this change will be a bit more involved to get support for Windows SDK, maybe it would be better to be in an additional PR after this one and have someone that works on the Windows platform to help enable that support, I do not have the ability to test on Windows. But if anyone knows how to add support and has the ability to test it out, please let me know and I can add it to this PR!